### PR TITLE
NOISSUE - Add linters

### PR DESCRIPTION
**Pull request by SammyOina**
Add linters
- switchTrue
- importShadow
-  httpNoBody 
- paramTypeCombine 
- emptyStringTest  
- builtinShadow  
- exposedSyncMutex  
- importas

### CODE REVIEW

File: internal/libvirt/libvirt.go [hunk 1]
```diff
@@ -10,12 +10,12 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/digitalocean/go-libvirt"
+	golibvirt "github.com/digitalocean/go-libvirt"
```
Comment (rodneyosodo) [inline internal/libvirt/libvirt.go:13]: Why import aliasing for this? If it is necessary 
```suggestion
	libvirt "github.com/digitalocean/go-libvirt"
```

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: internal/libvirt/libvirt.go (hunk 1 above)
Comment (SammyOina) [reply internal/libvirt/libvirt.go:13]: I use golibvirt to avoid an import shadow

**Review by rodneyosodo — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**